### ISSUE TITLE
Question about the clip_max_norm.

### ISSUE THREAD

**Issue by yelantingfeng** [facebookresearch/detr#198]
Hello, thanks for sharing this amazing work! I have a question for the argument `clip_max_norm`. 

I tried training DETR on COCO dataset and inspected the gradient norm at early iterations. The magnitude of gradient norm I observed are in the range 10^1~10^2, while in DETR we use `clip_max_norm=0.1`. I wonder why is this hyper-parameter set so low and could you please give me some suggestions about gradient clipping (in more general cases)?

Many thanks in advance!

**Comment by yelantingfeng** [comment]
I found this is duplicate to #159 .
